### PR TITLE
Pcx 428 change button label to ok

### PR DESCRIPTION
**Pull request by siebeprojects**

### CODE REVIEW

File: payment/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -19,5 +19,5 @@
     <string name="pmlocal_button_cancel">Cancel</string>
     <string name="pmlocal_button_retry">Retry</string>
     <string name="pmlocal_button_update">Update</string>
-    <string name="pmlocal_button_ok">OK</string>
+    <string name="pmlocal_button_ok">Ok</string>
```
Comment (hjhill) [inline payment/src/main/res/values/strings.xml]: Are you sure it should be "Ok" and not "OK"? I've never seen it this way...

Comment (siebeprojects) [reply payment/src/main/res/values/strings.xml]: According to my designer yes, it is "Ok". 
And so does this designer website use:
https://uxmovement.com/buttons/why-ok-buttons-in-dialog-boxes-work-best-on-the-right/

Comment (darklynx) [reply payment/src/main/res/values/strings.xml]: @siebeprojects the very last comment of on the page refer is contradicting to yours statment :)

Comment (darklynx) [reply payment/src/main/res/values/strings.xml]: Mac OS designers are also disagree: 
Screenshot 2020-03-18 at 20 33 03

Comment (darklynx) [reply payment/src/main/res/values/strings.xml]: Also: https://www.google.com/search?q=iPhone+dialogs+with+OK&tbm=isch

**Pull request merged**